website/content/docs/concepts/events.mdx

```diff
@@ -131,6 +137,13 @@ Here is an example event notification in JSON format:
 
 ## Subscribing to event notifications
 
+<Note title="Events can only be subscribed from the active node">
+
+For multi-node vault deployments, currently events can only be subscribed from the active node. If events are subscribed from a standby
+node the websocket connection will be redirected to the active node using the `api_addr` from the active node's Vault configuration.
+
+</Note>
```

Comment:
We should probably capitalize "vault" in the first sentence, and may want to write in more active voice, like "For multi-node Vault deployments, Vault only accepts event subscriptions on the active node. If a client attempts to subscribe to events on a standby node, Vault will respond with a redirect to the active node..."

It also might be useful to link to the `api_addr` parameter in the docs (`/vault/docs/configuration#api_addr`).

We'll also probably want to rework this a bit since https://github.com/hashicorp/vault/pull/28853 has been merged, and adds a similar note in the same spot.

Reply:
thanks, I made changes here accordingly